Libraries/LibVT/Terminal.h

```diff
@@ -148,6 +148,8 @@ class Terminal {
 
     void emit_string(const StringView&);
 
+    void alter_mode(bool, bool, const ParamVector&);
```

Comment:
Please don't omit parameter names where it's not obvious from context/type what the parameter does :)